quilt/tools/store.py

```diff
@@ -315,6 +316,7 @@ def _add_to_contents(self, fullname, objhash, ext, path, target):
         leaf = ipath.pop()
 
         ptr = contents
+        ptr.setdefault(TYPE_KEY, NodeType.GROUP.value)
```

Comment:
Why not just make sure `get_contents` always returns a dict with the key? If the package doesn't have any objects, this won't run, right?